src/wfs/src/main/java/org/geoserver/wfs/xml/xs/DateBinding.java

```diff
@@ -17,13 +17,24 @@
  * timezone.
  *
  * @author Justin Deoliveira, The Open Planning Project
- *
+ * @author awaterme
  */
 public class DateBinding extends XSDateBinding {
     public String encode(Object object, String value) throws Exception {
+    
+		/*
+		 * GEOS-1235, GEOS-6777, GEOS-3651, GEOS-7053:
+		 * The string conversion is implementation by
+		 * org.geotools.xml.impl.XsDateTimeFormat#format(Object, StringBuffer,
+		 * java.text.FieldPosition): Apparently the CITE tests require a UTC
+		 * Timezone. To avoid flipping into the previous or next day, depending
+		 * on the system default time zone, consider the offset.
+		 */
```

Comment:
GeoServer uses 4 spaces instead of tabs for indentation. In the interests of keeping code style as consistent as possible (and avoiding files that mix spaces and tabs), please convert all tabs to spaces.

For more information about GeoServer code formating and other policies, see the [Developer Manual](http://docs.geoserver.org/stable/en/developer/index.html) especially the [submitting patches](http://docs.geoserver.org/stable/en/developer/policies/patches.html#ensure-your-ide-editor-is-properly-configured) section.